### PR TITLE
Indicate that a user has selected some datagrid items

### PR DESCRIPTION
**Pull request by Kemosabert**
### Description
If a Datagrid's items are selectable, it will now also show when some items have been selected by showing a "-" icon in the checkbox as indicator to the user.

#### Screenshot before this PR
screen shot 2018-07-06 at 11 34 14

#### Screenshot after this PR
screen shot 2018-07-06 at 11 35 41

### Breaking changes
none

### CODE REVIEW

**Comment by lowiebenoot** [comment]
Maybe add a story for it under checkbox as well.

File: components/checkbox/Checkbox.js [hunk 1]
```diff
@@ -98,7 +98,11 @@ class Checkbox extends PureComponent {
           {...inputProps}
         />
         <span className={theme['shape']}>
-          <IconCheckmark className={theme['checkmark']} />
+          {partiallySelected ? (
```
Comment (lowiebenoot) [inline components/checkbox/Checkbox.js]: I would turn them around.
If you set both `partiallySelected` and `checked` to true, I would expect `checked` to get the upper hand, and show a checked checkbox.

Comment (driesd) [reply components/checkbox/Checkbox.js]: Damn good catch!

File: components/checkbox/Checkbox.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ class Checkbox extends PureComponent {
   }
 
   render() {
-    const { checked, disabled, className, size, label, children, ...others } = this.props;
+    const { checked, disabled, className, size, label, children, partiallySelected, ...others } = this.props;
```
[the remark this answers is not in the document]
Comment (Kemosabert) [reply components/checkbox/Checkbox.js]: That is actually very true, the variable here is poorly named as we describe a state of the checkbox itself, not what is happening around it. I think "indeterminate" is a perfect word/name for it, but also not sure how clear this is for others. What do you think about this?

File: components/checkbox/Checkbox.js (hunk 1 above)
Comment (Kemosabert) [reply components/checkbox/Checkbox.js]: Good catch, although I already made sure in the logic in the datagrid component that both checked  and partiallyChecked can not be true at the same time.

 :ok: by @Kemosabert on 06/07/2018 15:45:49

File: components/checkbox/Checkbox.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (lowiebenoot) [reply components/checkbox/Checkbox.js]: `indeterminate` seems good to me. It's not the clearest name, but as it is documented, people will know what it means (and people will probably don't need it very often).

**Pull request merged**